tray/src/com/dvt/main/creational/prototype/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+<<<<<<< HEAD
+package com.dvt.creational.prototype;
+=======
+package creational.prototype;
+>>>>>>> main
```

Comment:
Please fix merge conflict.
```suggestion
<<<<<<< HEAD
package com.dvt.creational.prototype;
=======
package creational.prototype;
>>>>>>> main
```